mod/reportedcontent/classes/Elgg/ReportedContent/Notifications/CreateReportedContentNotificationHandler.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Elgg\ReportedContent\Notifications;
+
+use Elgg\Notifications\NotificationEventHandler;
+use Elgg\Database\QueryBuilder;
+
+class CreateReportedContentNotificationHandler extends NotificationEventHandler {
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getSubscriptions(): array {
+		$admins = elgg_get_entities([
+			'type' => 'user',
+			'limit' => false,
+			'batch' => true,
+			'metadata_name_value_pairs' => [
+				[
+					'name' => 'admin',
+					'value' => 'yes',
+				],
+			],
+			'private_setting_name_value_pairs' => [
+				[
+					'name' => 'admin_notification',
+					'value' => 1,
+				],
+			],
+			'private_setting_name_prefix' => 'plugin:user_setting:reportedcontent:',
+			'wheres' => [
+				function (QueryBuilder $qb, $main_alias) {
+					// no idea why an admin would report content and not fix it themselfs, but just in case
+					return $qb->compare("{$main_alias}.guid", '!=', $this->event->getActorGUID(), ELGG_VALUE_GUID);
+				}
+			],
+		]);
+		
+		$subscriptions = [];
+		foreach ($admins as $admin) {
+			$subscriptions[$admin->guid] = ['email'];
+		}
+		
+		return $subscriptions;
+	}
+	
+	/**
+	 * {@inheritDoc}
+	 */
+	public static function isConfigurableByUser(): bool {
+		return false;
```

Comment:
why not have this as a notification preference again (so not a configurable event, but as a 'default' preference)

Reply:
This only applies to detailed notification setting. This is not something which can be configured on an entity level

Reply:
changed it to a generic notification setting